db/postgres_schema.rb

```diff
@@ -472,7 +473,7 @@
     t.index ["owner_type", "owner_id"], name: "index_deleted_objects_on_owner_type_and_owner_id"
   end
 
-  create_table "email_configurations", force: :cascade do |t|
+  create_table "email_configurations", id: :serial, force: :cascade do |t|
```

Comment:
any idea how this changed?

Reply:
Oh...

This had happened to me before, and I didn't commit these changes previously.
I am not sure whether it's due to some minor change in Rails version or what. But I'll revert it I guess.

Reply:
I reverted. If `id: :serial` is really needed in this case, I guess it will be fixed in Rails 6 upgrade.